### PR TITLE
【MetaCLIP】添加MetaCLIP模型paddle版本

### PR DESCRIPTION
**Pull request by ssstuding**
【MetaCLIP】添加 MetaCLIP_B_16 模型 Paddle 版本
## 模型介绍
添加 MetaCLIP_B_16 的 PaddlePaddle / PaddleClas 实现。
MetaCLIP 是基于 CLIP 的视觉编码器，通过元数据驱动的数据管理策略，在 CLIP 训练中实现了更好的数据质量和模型性能。

### 权重文件
MetaCLIP权重已上传至 AI Studio 星河社区：

https://aistudio.baidu.com/modelsdetail/51427/space

## 验证内容

- [x] 前向对齐验证：与 PyTorch 版本输出对齐，误差 < 1e-4
- [x] 可训练性验证：Loss 可正常下降
- [x] 性能验证：静态图加速达标

### 前向对齐验证

| 验证项 | 结果 | 说明 |
|--------|------|------|
| Max Absolute Diff | < 1e-4 | 与 PyTorch 版本对齐 |
| Avg Absolute Diff | < 1e-6 | 与 PyTorch 版本对齐 |
| 对齐状态 | ✅ 通过 | 前向输出一致 |

### 可训练性验证

| 验证项 | 结果 | 说明 |
|--------|------|------|
| 初始 Loss | 正常 | 模型初始化正确 |
| Loss 下降 | ✅ 正常 | 梯度回传正常 |
| 训练稳定性 | ✅ 稳定 | 无 NaN/Inf |

### 性能验证

| 验证项 | 结果 | 说明 |
|--------|------|------|
| 动态图推理 | ✅ 正常 | - |
| 静态图推理 | ✅ 正常 | - |
| 静态图加速 | > 10% | 达标 |

### 验证结果的截图：
1BD33894E59BE5A208F4ADEA436E0D35

## 文件变更

| 文件 | 说明 |
| --- | --- |
| `ppcls/arch/backbone/model_zoo/metaclip.py` | MetaCLIP_B_16 模型实现 |
| `ppcls/arch/backbone/__init__.py` | 注册 MetaCLIP_B_16 模型 |
| `ppcls/configs/MetaCLIP/metaclip_base.yaml` | MetaCLIP_B_16 配置文件 |
| `docs/zh_CN/models/MetaCLIP/metaclip_base.md` | 模型文档与权重说明 |

### CODE REVIEW

File: ppcls/configs/MetaCLIP/metaclip_base.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Arch:
```
Comment (zhangyubo0722) [inline ppcls/configs/MetaCLIP/metaclip_base.yaml:33]: 配置文件需要写完整，包括数据部分，学习率部分，支持完整的训练流程。

File: ppcls/arch/backbone/model_zoo/metaclip.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import paddle
```
Comment (zhangyubo0722) [inline ppcls/arch/backbone/model_zoo/metaclip.py:1]: 该模型权重链接需要添加，把权重发给负责人，负责人上传后提供链接

File: ppcls/arch/backbone/model_zoo/metaclip.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import paddle.nn as nn
+import paddle.nn.functional as F
+from functools import partial
+
+print("🔍 metaclip_paddle.py is being loaded...")
```
Comment (zhangyubo0722) [inline ppcls/arch/backbone/model_zoo/metaclip.py]: 去掉没必要的注释

File: ppcls/arch/backbone/model_zoo/metaclip_verify/align_metaclip.py [hunk 4]
```diff
@@ -0,0 +1,228 @@
+import torch
```
Comment (zhangyubo0722) [inline ppcls/arch/backbone/model_zoo/metaclip_verify/align_metaclip.py]: metaclip_verify文件夹可以额外提供，但是不要提交pr

File: ppcls/configs/MetaCLIP/metaclip_base.yaml (hunk 1 above)
Comment (ssstuding) [reply ppcls/configs/MetaCLIP/metaclip_base.yaml:33]: 已补全 metaclip_base.yaml，增加 Global、AMP、Loss、Optimizer、DataLoader、Infer、Metric 等完整训练/验证/推理配置，并补充预训练权重和数据路径说明。

File: ppcls/arch/backbone/model_zoo/metaclip.py (hunk 2 above)
Comment (ssstuding) [reply ppcls/arch/backbone/model_zoo/metaclip.py:1]: 已将权重上传至 AI Studio 星河社区：
https://aistudio.baidu.com/modelsdetail/51427/space

其中包含转换后的 PaddlePaddle 权重 metaclip_b16.pdparams 和原始 PyTorch/timm 权重 metaclip_b16.safetensors。

File: ppcls/arch/backbone/model_zoo/metaclip.py (hunk 3 above)
Comment (ssstuding) [reply ppcls/arch/backbone/model_zoo/metaclip.py]: 已清理 metaclip.py 中不必要的调试输出、无用 import 和注释。

File: ppcls/arch/backbone/model_zoo/metaclip_verify/align_metaclip.py (hunk 4 above)
Comment (ssstuding) [reply ppcls/arch/backbone/model_zoo/metaclip_verify/align_metaclip.py]: 已移除 metaclip_verify 验证脚本目录，不再提交到 PR。

File: ppcls/arch/backbone/model_zoo/metaclip.py (hunk 2 above)
Comment (ssstuding) [reply ppcls/arch/backbone/model_zoo/metaclip.py:1]: 已根据反馈更新：

1. 模型名称和权重名称已统一为 `MetaCLIP_B_16` / `MetaCLIP_B_16_pretrained.pdparams`。
2. 权重已使用 AI Studio Git LFS 命令行方式重新上传。
3. AI Studio 模型空间中已删除旧的 `metaclip_b16.pdparams` 和 `metaclip_b16.safetensors`，仅保留核心 Paddle 权重。
4. PR 中代码、配置和文档已同步修改。

权重页面：https://aistudio.baidu.com/modelsdetail/51427/space

验证：
- MetaCLIP_B_16 模型构造测试通过
- `git diff --check` 无格式问题
